web/src/components/Features.js

```diff
@@ -103,29 +104,28 @@ const Features = () => {
 
         <Feature
           Icon={Send}
-          title="Simple Deployment"
+          title="One-command Deploy"
           url="/docs/deployment/intro"
           description={`
-            Deploy your app to any platform. Wasp offers CLI helpers for the most popular options.
+            Deploy your entire app—from frontend to database—to platforms like Railway or Fly.io with just one command.
           `}
         />
 
         <Feature
-          Icon={Settings}
-          title="Jobs"
-          url="/docs/advanced/jobs"
+          Icon={Mail}
+          title="Email Sending"
+          url="/docs/advanced/email"
           description={`
-            Easily define, schedule and run specialized server tasks.
-            Persistent, retryable, delayable.
+            Just choose a provider and Wasp will handle the rest!
           `}
         />
 
         <Feature
-          Icon={Mail}
-          title="Email Sending"
-          url="/docs/advanced/email"
+          Icon={Settings}
+          title="Jobs"
+          url="/docs/advanced/jobs"
           description={`
-            All you need to do is connect an email provider and you can send emails!
+            Easily send emails, crunch data, and schedule tasks using the power of Postgres.
```

Comment:
Hm I found the earlier desription clearer. I think you focused more on what you can do with it, but now it is less clear to me what it is, is it something specific for emails/data/scheudling or is it a general mechanism?